scripts/bind/pic/template.hbs

```diff
@@ -7,8 +7,11 @@ use pocket_ic::PocketIc;
 
 use crate::utils::pic_canister::{PicCanister, PicCanisterTrait};
 
-{{type_defs}}
+{{#if submodule}}
+pub mod {{submodule}};
+{{/if}}
```

Comment:
This optionally adds a submodule, that can be used to add methods to the auto-generated types.  I would prefer this to be an array but the didc toml only supports strings as template vars.  And de-facto one submodule is plenty for our use case.